### PR TITLE
fix: gnomad_re should accept all nucleotide characters for ref/alt

### PR DESCRIPTION
**Pull request by korikuzma**
Address #275 for main branch

### CODE REVIEW

**Review by toneillbroad — APPROVED** [review]

**Review by ahwagner — APPROVED** [review]
VRS supports all IUPAC codes, this allows all IUPAC codes. Seems reasonable to me!

File: src/ga4gh/vrs/extras/translator.py [hunk 1]
```diff
@@ -41,7 +41,10 @@ class Translator:
     """
 
     beacon_re = re.compile(r"(?P<chr>[^-]+)\s*:\s*(?P<pos>\d+)\s*(?P<ref>\w+)\s*>\s*(?P<alt>\w+)")
-    gnomad_re = re.compile(r"(?P<chr>[^-]+)-(?P<pos>\d+)-(?P<ref>[ACGTN]+)-(?P<alt>[ACGTN]+|\*|\.)", re.IGNORECASE)
+    gnomad_re = re.compile(
+        r"(?P<chr>[^-]+)-(?P<pos>\d+)-(?P<ref>[ACGTURYKMSWBDHVN]+)-(?P<alt>[ACGTURYKMSWBDHVN]+|\*|\.)",
```
Comment (ahwagner) [inline src/ga4gh/vrs/extras/translator.py:45]: @korikuzma you pointed out that we also use `*` or `.` here. I wouldn't expect to see that in gnomAD, and think we would need to build out advanced methods for handling that in our VCF processor. Are these characters used in gnomAD alt expressions?

```suggestion
        r"(?P<chr>[^-]+)-(?P<pos>\d+)-(?P<ref>[ACGTURYKMSWBDHVN]+)-(?P<alt>[ACGTURYKMSWBDHVN]+)",
```

Comment (ahwagner) [reply src/ga4gh/vrs/extras/translator.py:45]: I know these characters are part of VCF, but that is a file format convention specific to VCF; I do not think this is used by gnomAD, and as far as I can tell isn't required to address #275. So, if we do not have example records from gnomAD where this is needed, I propose we do not add `.` or `*` to the `gnomad_re` method. Our VCF translator _may_ need to handle these at a future date, but until that feature is requested by the community it is not a priority.

Comment (korikuzma) [reply src/ga4gh/vrs/extras/translator.py:45]: @ahwagner works for me! I didn't know if the gnomad_vcf was a combination of gnomad and vcf spec or not.

**Review by ahwagner — APPROVED** [review]

File: src/ga4gh/vrs/extras/translator.py (hunk 1 above)
Comment (korikuzma) [reply src/ga4gh/vrs/extras/translator.py]: @ahwagner I'm going to check with the gnomad team on whether `.` exist in their dataset.

**Comment by korikuzma** [comment]
Confirmed with gnomAD team that ALT will never be `.`.

**Pull request merged**